### PR TITLE
yarn workspaces

### PR DESCRIPTION
**Pull request by jamestalton**
Signed-off-by: James Talton <jtalton@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,9 +40,6 @@ Notes:
 
 - Workspaces are defined in the `package.json` at the root
 - While workspaces allow the sequential running of commands across all workspaces, console is using a project `workspaces-run` to run command across all workspaces concurrently.
-- Installing a new dependency for a workspace must be done at the root using the `--workspace` parameter.
-  - `npm install some-lib --workspace frontend`
-  - This is because npm hoists up the dependencies and updates the root package-lock.json.
```
Comment (KevinFCormier) [inline README.md:47]: Is there no similar restriction with yarn?

Comment (jamestalton) [reply README.md:47]: i think yarn takes care of it for you

**Review by KevinFCormier — DISMISSED** [review]
👍

**Pull request merged**